snackbars/src/main/res/values/styles-snackbar.xml

```diff
@@ -13,6 +13,15 @@
         <item name="android:paddingBottom">@dimen/vtmn_snackbars_padding_vertical</item>
```

Comment:
<item name="android:paddingEnd">@dimen/vtmn_snackbars_padding_end</item>

Add a padding end to the text so that it is aligned with button 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/46710837/236232139-f0ab4740-af26-49f3-8b9c-00511bd68e36.png">